processor/redactionprocessor/README.md

```diff
@@ -111,6 +115,11 @@ part of the value is not masked even if it matches the regular expression for a
 If the value matches the regular expression for a blocked value only, the matching
 part of the value is masked with a fixed length of asterisks.
 
+`hash_function` defines the function for hashing the values (or substrings of values)
+instead of masking them with a fixed string. By default, no hash function is used
+and masking with a fixed string is performed. The supported hash functions
+are `md5`, `sha1` and `sha3`.
```

Comment:
Is it worth mentioning what specific variant of these is implemented? (e.g. is it SHA-256 or SHA-512?)

Reply:
I would say it's good this way

Reply:
I would at least mention it for SHA3, since Go allows you to choose the size